<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddViewDialog</class>
 <widget class="QDialog" name="AddViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ajouter un visionnage</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="3">
    <spacer name="LowerVerticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="MovieViewDateLabel">
     <property name="text">
      <string>Date de visionnage</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="MovieNameLabel">
     <property name="text">
      <string>Nom du film</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="MovieRatingLabel">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="UpperVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="MovieReleaseYearLabel">
     <property name="text">
      <string>Année de sortie</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="MovieViewTypeLabel">
     <property name="text">
      <string>Type de visionnage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="EntriesLabel">
     <property name="text">
      <string>Entrées</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="PosterTextLabel">
     <property name="text">
      <string>Affiche</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QSpinBox" name="MovieRatingInput">
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QSpinBox" name="EntriesInput">
     <property name="maximum">
      <number>100000000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QSpinBox" name="MovieReleaseYearInput">
     <property name="minimum">
      <number>1870</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>1900</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="MovieNameInput"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="NewMovieLabel">
     <property name="text">
      <string>où ajoutez un nouveau film</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="ExistingMoviesComboBox">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="SelectMovieLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Selectionnez un film déjà vu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QPushButton" name="PosterButton">
     <property name="text">
      <string>Sélectionnez une image</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QDateEdit" name="MovieViewDateInput">
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>7</month>
       <day>4</day>
      </datetime>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>9999</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1899</year>
       <month>12</month>
       <day>29</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QComboBox" name="ViewTypeComboBox">
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Blu-Ray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cinéma</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DVD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Streaming</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Téléchargement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VHS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Autre support physique</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QCheckBox" name="UnknownViewTypeCheckbox">
     <property name="text">
      <string>inconnu</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QCheckBox" name="UnknownViewDateCheckbox">
     <property name="text">
      <string>inconnu</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="Line" name="SeparationLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="15" colspan="2">
    <widget class="QLabel" name="PosterLabel">
     <property name="text">
      <string>Affiche</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddViewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddViewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
